### ISSUE TITLE
Release latest version?

### ISSUE THREAD

**Issue by ivoanjo** [garmoshka-mo/pry-moves#15]
Howdy :wave: 

Thanks a lot for working on `pry-moves`! I myself submitted a few patches to `pry-nav` in the past, but unfortunately it was really had to get them merged.

I've noticed that latest `master` includes quite a few advances/fixes when compared to the latest released version. Have you considered releasing them on rubygems.org?

Have an awesome weekend!

**Comment by garmoshka-mo** [comment]
Yes, though there is so many changes in logic and some dependencies, that new version can be buggy on random new project. 
Can you please test gem from github master branch (`gem 'redcarpet', :git => 'git://github.com/garmoshka-mo/pry-moves.git'`) and give feedback - if everything works / any bugs appear for you?

**Comment by ivoanjo** [comment]
Seems to work reasonably, I tried it with the specs on my [persistent-dmnd](https://gitlab.com/ivoanjo/persistent-dmnd/) gem. I do get quite a few warnings though:

```
rvm/gems/ruby-2.7.2/bundler/gems/pry-moves-7cb3e4c94a78/lib/pry-moves/bindings_stack.rb:47: warning: in `eval'
rvm/gems/ruby-2.7.2/bundler/gems/pry-moves-7cb3e4c94a78/lib/pry-moves/backtrace.rb:8: warning: instance variable @filter not initialized
rvm/gems/ruby-2.7.2/gems/rspec-core-3.10.1/exe/rspec:4: warning: __FILE__ in eval may not return location in binding; use Binding#source_location instead
```

**Comment by garmoshka-mo** [comment]
Thank you for details. I have ruby warnings disabled globally on my machine with `export RUBYOPT="-W0"` 

I've started to fix code, but apparently there is also warning for gem dependencies code (e.g. `pry-0.12.2/lib/pry/method/weird_method_locator.rb:28: warning: in `eval'`) 

I never had issues caused by ignoring warnings, thus I disabled warnings for pry-moves.
Could you please re-test gem with latest commits?

**Comment by ivoanjo** [comment]
Thanks again for looking into this :)

> I've started to fix code, but apparently there is also warning for gem dependencies code (e.g. pry-0.12.2/lib/pry/method/weird_method_locator.rb:28: warning: in eval'`)

I'm guessing the errors may be fixed in newer pry versions, some of the warnings may have been introduced in the meanwhile.

Ok, so I've given it a bit more testing, and saw a few things:

1. Running `bundle exec pry` prints part of the pry cli context. I'm guessing it should autodetect when it's being started from the cli and not print itself.

2. `binding_of_caller` 0.8.0 breaks on Ruby 3. Seems ok for 2.7, but when I tried switching rubies it just completely broke.

**Comment by garmoshka-mo** [comment]
Just tested previous versions of `pry-moves` it doesn't work with ruby 3 as well, restricted ruby version for gem to be < 3

**Comment by garmoshka-mo** [comment]
Published 1.0.0 to rubygems

Issue with `bundle exec pry` seems like minor, though feel free to submit PR if you wish.
Thank you for help in testing

**Comment by ivoanjo** [comment]
Thanks for working on this! :)
